tools/tsp-client/src/index.ts

```diff
@@ -219,18 +219,6 @@ const parser = yargs(hideBin(process.argv))
       await sortSwaggerCommand(argv);
     },
   )
-  .command(
```

Comment:
How about the [compare command](https://github.com/Azure/azure-sdk-tools/blob/8154119349eb9fe5d6db5f45df2eae30f6aa1b9f/tools/tsp-client/test/run_commands.ts#L83) being called in the test? Will the test run be part of the CI?

Reply:
closing this pr in favor of https://github.com/Azure/azure-sdk-tools/pull/8811